.circleci/config.yml

```diff
@@ -15,15 +15,14 @@ references:
   images:
     # When updating the Go version, remember to also update the versions in the
     # workflows section for go-test-lib jobs.
-    go: &GOLANG_IMAGE docker.mirror.hashicorp.services/circleci/golang:1.17.2
+    go: &GOLANG_IMAGE docker.mirror.hashicorp.services/cimg/go:1.17.5
     ember: &EMBER_IMAGE docker.mirror.hashicorp.services/circleci/node:14-browsers
 
   paths:
     test-results: &TEST_RESULTS_DIR /tmp/test-results
 
   cache:
     yarn: &YARN_CACHE_KEY consul-ui-v6-{{ checksum "ui/yarn.lock" }}
-    rubygem: &RUBYGEM_CACHE_KEY static-site-gems-v1-{{ checksum "Gemfile.lock" }}
```

Comment:
Why this need to be removed ? Is something different about the docker image we are using?

Reply:
The rubygem cache was being unused in the file so I just deleted it